cmd/fyne_demo/tutorials/bind.go

```diff
@@ -82,7 +83,11 @@ func newFormWithData(data binding.DataMap) *widget.Form {
 	keys := data.Keys()
 	items := make([]*widget.FormItem, len(keys))
 	for i, k := range keys {
-		items[i] = widget.NewFormItem(k, createBoundItem(data.GetItem(k)))
+		data, ok := data.GetItem(k)
+		if !ok {
+			continue
```

Comment:
Can form deal with one of the items being nil?

Reply:
no :(